icu4j/main/core/src/test/java/com/ibm/icu/dev/test/TestBoilerplate.java

```diff
@@ -39,6 +39,7 @@ public abstract class TestBoilerplate<T> extends TestFmwk {
 
     protected static Random random = new Random(12345);
 
+    @SuppressWarnings({"SelfEquals", "EqualsNull"})
```

Comment:
`EqualsNull` & `SelfEquals`

I was tempted to delete them. They seem kind of superfluous.
But they are in the test freamework, they don't affect prod code.
And at some point someone thought they are a good idea. I don't know why, but they don't hurt.

Reply:
Thumb up means "it's good that you suppressed the warnings" or "I would be OK if you remove those tests" :-)

Reply:
I think it makes sense that you suppressed these warnings here given the apparent intent of the tests.
They complain about Java invariants that we are double-checking hold true for our code.

Reply:
Got it. Thanks, I'll submit.